frontend/apps/reader/modules/readergoto.lua

```diff
@@ -60,7 +47,7 @@ function ReaderGoto:onShowGotoDialog()
         input_hint = T("@%1 (1 - %2)", curr_page, self.document:getPageCount())
     end
     self.goto_dialog = InputDialog:new{
-        title = dialog_title,
+        title = _("Go to Page"),
```

Comment:
Should that be `Go to page`? I think we might have some inconsistencies.

Reply:
You tell me :) We have a few (mostly in the bottom menu) `Camel Titles`.
But ok, looks ugly in dialog titles.

Reply:
I'd prefer to use `Go to page`.